Dannyyoung20/fast-food-fast
# PR fixes the set headers bug

#### What does this PR do?
PR fixes the set headers bug
#### Description of Task to be completed?
PR fixes the set headers bug
#### How should this be manually tested?
Create a function that queries our db to verify email and add it to our login function which also queries or db
#### What are the relevant pivotal tracker stories?
#160901914
#### Any background context you want to add?

#### Screenshots
